MovieQuiz/Presentation/AlertPresenter.swift

```diff
@@ -0,0 +1,21 @@
+import UIKit
+
+class AlertPresenter {
+    private weak var alertPresenter: MovieQuizViewController?
+    init(alertPresenter: MovieQuizViewController?){
```

Comment:
давай договоримся, между разными сущностями оставлять одну пустую строчку  (между перечислением свойств и функцией, между функциями, между типами)
```suggestion
    private weak var alertPresenter: MovieQuizViewController?
    
    init(alertPresenter: MovieQuizViewController?){
```